src/browser/modules/Stream/CypherFrame/relatable-view.tsx

```diff
@@ -91,7 +103,10 @@ function getColumns(records: any, maxFieldItems: any) {
   }))
 }
 
-function CypherCell({ cell }: any) {
+type CypherCellProps = {
+  cell: any
```

Comment:
I tried to do that, but I basically ended up with almost any type, since StringifyMod accepts basically anything. Probably we need to rewrite the code to make it more explicitly clear what types that are accepted here but I do feel that it requires someone with more insight into the code at this moment.